src/publish/confluence/confluence-helper.ts

```diff
@@ -624,17 +634,22 @@ export const findAttachments = (
 
   if (publishFiles.length > 0) {
     uniqueResult = uniqueResult.map((assetFileName: string) => {
-      const assetInPublishFiles = publishFiles.find((assetPath) => {
-        return assetPath.endsWith(`${parentPath}/${assetFileName}`);
+      const assetInPublishFiles = publishFiles.find((assetPathParam) => {
+        const assetPath = assetPathParam.replaceAll("\\", "/");
```

Comment:
We do have a function which essentially does the same thing which probably makes sense to use:

`pathWithForwardSlashes`

Reply:
oo thank you

Reply:
replaced